### DISCUSSION TITLE
NodeMCU ESP8266 de 32MB

### DISCUSSION THREAD

**Discussion by Piero15** [esp8266/Arduino#8281] [Q&A] [answered]
hola es la primera vez que uso esta tarjeta de desarrollo, y desde ya tengo problemas no encuentro por ningún lado como poder programarlo correctamente ya que mi tarjeta es de 32MB memory flash, y las configuraciones en la tarjeta solo llega hasta 16MB, asi que recurro a ustedes por una ayuda por favor ;c

**Comment by earlephilhower** [comment, marked as answer]
Tu ESP8266 no tiene que 4 *megabytes*.  4 mega*bytes* es iqual a 32 mega*bits*.  Se vende "32mb" porque parece mas largo...

**Reply by Piero15** [reply]
entiendo....., muchas gracias por la ayuda, y perdon por la ignoracia

entonces lo correcto seria configurarlo a 4MB verdad?

**Reply by earlephilhower** [reply]
Si
